tests/Microsoft.Identity.Test.Core.UIAutomation/IOSXamarinUITestController.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xamarin.UITest.Queries;
+
+namespace Microsoft.Identity.Test.UIAutomation.Infrastructure
+{
+    public class IOSXamarinUITestController : XamarinUITestController
```

Comment:
can you rename UI to Ui in both namespaces?

Reply:
done